src/ostorlab/apis/scanner_config.py

```diff
@@ -0,0 +1,56 @@
+"""Receive scanner config (Nats creds, Harbor creds ...)."""
+
+from typing import Dict, Optional
+import json
+
+from ostorlab.apis import request
+
+
+class GetScannerConfigAPIRequest(request.APIRequest):
+    """Get scanner config."""
+
+    def __init__(self, scan_id: int):
+        self._scan_id = scan_id
+
+    @property
+    def query(self) -> Optional[str]:
+        """Defines the query to get the configs.
+
+        Returns:
+            The query to get the configs.
+        """
+        return """
+         query scanners($scannerId: Int){
```

Comment:
Is the scannerId parameter required ? In that case, can you please add the `!` mark at the end.